zig-mode.el

```diff
@@ -150,6 +153,8 @@ If given a SOURCE, execute the CMD on it."
              (when zig-format-show-buffer
                (progn
                  (pop-to-buffer fmt-buffer)
+		 (when zig-ansi-color-for-format-errors
+		   (ansi-color-apply-on-region (point-min) (point-max)))
```

Comment:
This is a nitpick, but could you indent this with `indent-tabs-mode` off? Would be nice if it looked on GitHub as nice as it would in Emacs.

Reply:
Happy to do so.

Reply:
Thanks!